src/content/docs/recipes/continuous-integration.mdx

```diff
@@ -54,3 +54,34 @@ jobs:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review
 ```
+
+## GitLab CI
+
+Below is an example configuration:
+
+```yaml title=".gitlab-ci.yml"
+# Define pipeline stages
+stages:
+  - quality
+
+# Lint job configuration: runs code quality checks using Biome
+lint:
+    image: 
+      name: ghcr.io/biomejs/biome:latest  # Use the latest Biome Docker image
+      entrypoint: [""]                    # This is required for the image to work in GitLab CI
+    stage: quality                        # Run in the quality stage
+    script:
+        - biome ci src --reporter=gitlab --colors=off --log-level=none --diagnostic-level=error > code-quality.json  # Execute Biome on the 'src' directory
+    artifacts:
+        paths:
+            - code-quality.json    # Collect the code quality report as an artifact
+    needs: []                      # No dependent jobs needed
+    rules:
+        - if: $CI_COMMIT_BRANCH    # Run job for commits on branches
+        - if: $CI_MERGE_REQUEST_ID # Run job for merge requests
+    allow_failure: true            # Allow the job to fail (adjust as needed)
```

Comment:
```suggestion

```

Perhaps we remove this one too. I expect people who use GitLab CI on a daily basis to know about this option. It keeps the example leaner. What do you think ?